service/http.proxy/src/test/java/org/kaazing/gateway/service/http/proxy/SecureOriginServer.java

```diff
@@ -58,11 +58,14 @@ public void run() {
             try(SSLSocket acceptSocket = (SSLSocket) socket.accept()) {
                 try {
                     handler.handle(acceptSocket);
-                } catch (Exception ioe) {
+                } catch (IOException ioe) {
                     ioe.printStackTrace();
+                    if (this.ioe != null) {
+                        this.ioe = ioe;
+                    }
                 }
             } catch (IOException ioe) {
-                this.ioe = ioe;
+                // no-op since socket.accept() may throw IOE
```

Comment:
Please log this exception at the trace level.  No exception should ever go uncaught.